Crust Crawler Arms/read_joint_positions/read_joint_positions.ino

```diff
@@ -34,8 +34,10 @@ void loop(){
   // and send it to the PC.
   for(int i = 1; i < 6; ++i){
     int pos = GetPosition(i);
-    String report;
-    report = i + ": " + pos;
+    String servo=String(i);
+    String text=String(": ");
+    String posit=String(pos);
+    String report = String(servo+text+posit);
```

Comment:
The syntax is obviously amateur. My point was that it couldn't concatenate the int types with the string ": ". I also found [this site](https://www.arduino.cc/en/Tutorial/StringAdditionOperator). If you scroll a bit down it actually explains the same scenario of concatenation. It says "This won't compile because the compiler doesn't handle the operator precedence correctly." I use the 1.0.6 version for the Crustcrawler.

Anyway, I just changed the code on my computer. If you say it works for you than something must be wrong on my side, but I wrote the code line by line and the only time it couldn't compile was when I tried to create the report string.